packages/ts-types/README.md

```diff
@@ -37,11 +37,11 @@ Try it out now:
 
 For the latest version:
 
-- `npm install @openui5/ts-types --dev`
+- `npm install @openui5/ts-types --save-dev`
```

Comment:
maybe we should have a commit that fixes this mistake and a separate PR to fix the issues of JS + TS docs?

Reply:
or you simply reject this pull request and take over what you like manually.

Reply:
Thanks again for pointing out the incorrect argument I fixed it in:
- https://github.com/SAP/ui5-typescript/commit/21fb32da3d782b68b1a17ca7277bea3e8c20f775

I will try to find the time to refactor the documentation into two separate versions (TS/JS)